modals.js

```diff
@@ -1,4 +1,4 @@
-const helpModal = (screen, blessed) => {
+const helpModal = (screen, blessed, prog) => {
```

Comment:
Would be better for this not to need program passed, the callback may be cleaner - also, in any case, I would suggest a better name than prog